src/app/frontend/uicomponents/helpsection/helpsection.scss

```diff
@@ -0,0 +1,21 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.kd-help-section > * {
+  width: 50% !important;
```

Comment:
I think that flex needs to be defined for all direct children of `kd-help-section` to make it work. I've tried wrapping or defining only on one child but then second child is not flexed. Instead of `width: 50% !important` it can be just `flex: 1` so all children will have this attribute and can grow equally in available space.